docs/api/quickstart.md

```diff
@@ -43,14 +43,14 @@ Just let us know if you need help with a note via [email](mailto:hello@nearproto
 
 ### JSON-RPC
 - documentation is [here](/docs/interaction/rpc)
-- examples are [here](/docs/roles/developer/examples/nearlib/examples#jsonrpcprovider)
-- source code is [here](https://github.com/nearprotocol/nearlib/blob/master/src/providers/json-rpc-provider.ts)
-
-### JavaScript (`nearlib`)
-- documentation is [here](/docs/roles/developer/examples/nearlib/introduction)
-- examples are [here](/docs/roles/developer/examples/nearlib/examples)
-- source code is [here](https://github.com/nearprotocol/nearlib/tree/master/src)
-- tests are [here](https://github.com/nearprotocol/nearlib/tree/master/test)
+- examples are [here](/docs/roles/developer/examples/near-api-js/examples#jsonrpcprovider)
+- source code is [here](https://github.com/near/near-api-js/blob/master/src/providers/json-rpc-provider.ts)
+
+### JavaScript (`near-api-js`)
+- documentation is [here](/docs/roles/developer/examples/near-api-js/introduction)
+- examples are [here](/docs/roles/developer/examples/near-api-js/examples)
```

Comment:
I don't think these links are active until we merge this:
https://github.com/near/near-api-js/pull/281
And then either run the docs script and copy the files over to the docs repo, or use Chad's copy method here:
https://github.com/nearprotocol/docs/pull/291

Reply:
wait nevermind, disregard